build.sh

```diff
@@ -24,12 +24,8 @@ for d in $PLUGINS; do
 	if [ -d "$d" ]; then
 		plugin="$(basename "$d")"
 		echo "  $plugin"
-		# use go install so we don't duplicate work
-		if [ -n "$FASTBUILD" ]
-		then
-			GOBIN=${PWD}/bin go install -pkgdir $GOPATH/pkg "$@" $REPO_PATH/$d
-		else
-			go build -o "${PWD}/bin/$plugin" -pkgdir "$GOPATH/pkg" "$@" "$REPO_PATH/$d"
-		fi
+		go build -o ${PWD}/bin/$plugin -i -pkgdir "$GOPATH/pkg" "$@" "$REPO_PATH/$d" &
 	fi
 done
+
+wait
```

Comment:
The function of `wait` is described as follows in `man sh`:
> wait for all jobs to complete and the return an exit status of zero.

We would need to know if any of the background processes failed.
[Here](https://github.com/weaveworks/weave/blob/dff5b6d3f113105311fd5178ea5845150aba3360/test/setup.sh#L56) is an example how this can be done

Reply:
I've adjusted the script so it should do just that.